src/App.tsx

```diff
@@ -1,9 +1,21 @@
+import { RouterProvider, createBrowserRouter } from "react-router-dom";
 import SamplePage from "./pages/SamplePage";
+import Login from "./pages/Login";
 
 const App = () => {
+  const router = createBrowserRouter([
+    {
+      path: "/",
+      element: <SamplePage />
+    },
+    {
+      path: "login",
+      element: <Login />
+    }
+  ])
```

Comment:
So by this kind of implementation, it means that whenever there are new routes, we will have to add them here? What if you create a separate file, and then export an array of routes that will be imported here, so that whenever there are new routes created  they would be added in that array, instead of being added here?
I personnaly think that this way should be more convenient, than the  one implemented.

Reply:
I'll try to see how I can do it sir